client/src/pages/LogInForm.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+function LogInForm() {
+	const [emailInput, setEmailInput] = useState("");
+	const [passwordInput, setPasswordInput] = useState("");
+
+	// This variable handles shows success of the request to login with a boolean variable
+	const [backEndSuccess, setBackEndSuccess] = useState("");
+
+	const handleEmailChange = (e) => {
+		const email = e.target.value;
+		setEmailInput(email);
+	};
+
+	const handlePasswordChange = (e) => {
+		const password = e.target.value;
+		setPasswordInput(password);
+	};
+
+	async function postLogInDeatils(userData) {
+		try {
+			const response = await fetch("/mock-api", {
+				method: "POST",
+				headers: {
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify(userData),
+			});
+			const data = await response.json();
+			if (!response.ok) {
+				setBackEndSuccess(data.success);
+				throw new Error("Network response was not ok");
+			} else {
+				setBackEndSuccess(data.success);
+			}
+		} catch (error) {
+			setBackEndSuccess(error);
+		}
+	}
+	const handleFormSubmit = (e) => {
+		e.preventDefault();
+		const logInData = { email: emailInput, password: passwordInput };
+
+		postLogInDeatils(logInData);
+	};
+
+	return (
```

Comment:
I removed the part that threw an error and I use backEnd messgage instead. With this function, we can determine the appropriate message to display to the user based on the error they encounter. I hope this code works better.